src/component/mxgraph/shape/render/IconPainter.ts

```diff
@@ -354,7 +354,7 @@ export default class IconPainter {
   /**
    * This icon is used by `user task`.
    */
-  public paintUserIcon({ c, ratioFromParent, setIconOrigin, shape, icon }: PaintParameter): void {
+  public paintPersonIcon({ c, ratioFromParent, setIconOrigin, shape, icon }: PaintParameter): void {
```

Comment:
:question: why changing this one? I don't get what is the clarification/generalization here. It has already been renamed recently, see https://github.com/process-analytics/bpmn-visualization-js/pull/522/files#diff-5e09027c60c84ce5c53f738dd6ad77bbc15f103d8e0b99de725c8424d7cc2339R307
Be aware that changing this will break the custom icon example

Reply:
For me, it represents more a person  than a user. 
In my opinion, the term 'User' is just to add context to a task.

This icon can be used anywhere (but not sure if we re-use it), but not in the context of 'User'.

Good catch for the examples 😉

Reply:
for general context Person sounds better - so it depends if we want to reuse it elsewhere or just for task-icon

I would even propose to name it **paintFemalePersonIcon** as it actually draws a female :)

Reply:
We changed `paintWomanIcon` in `paintUserIcon` in the past, in order to allow to a developper to change woman to man.

In my opinion the icon painter should reflex only what it paints. And the class/method who uses it should add the context.
Maybe, an intermerdiate method is missing in the **task-shapes** file to call the `paintWomanIcon` (whatever its name). And the developper should override this missing method and not the woman methid if he want use a man icon.

Reply:
that makes sense 👍

Reply:
So, do I rename it in `paintWomanIcon` ?

Reply:
I meant ok for eventual intermediate method and overriding

here it is OK to stay with Person, for me, you can merge as it is (just remember about examples mentioned by Thomas)

Reply:
Done in https://github.com/process-analytics/bpmn-visualization-examples/pull/74